git/repo.go

```diff
@@ -86,7 +86,7 @@ func (r Repo) Clone(c Config) (*Checkout, error) {
 	}
 
 	notesRef, err := getNotesRef(repoDir, c.NotesRef)
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "Couldn't find remote ref") {
```

Comment:
`getNotesRef` doesn't try to fetch anything, so I'd be very surprised if it could result in that error (string). Does this definitely avoid a spurious error?